nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,651 @@
+    )
+
+    save_yhat0 = traits.Bool(
+        argstr="--save_yhat0", desc="save signal estimate (yhat)"
+    )
+
+    opt = traits.Int(
+        argstr="--opt %i",
+        desc="opt : optimization schema for applying adaptive GTM"
+    )  
+
+    opt_tol = traits.Tuple(
+        traits.Float,
+        traits.Float,
+        traits.Float,
+        argstr="--opt-tol %f %f %f",
+        desc=""
+    ) 
+
+    opt_brain = traits.Bool(
+        argstr="--opt-brain", desc="apply adaptive GTM"
+    ) 
+
+    opt_seg_merge = traits.Bool(
+        argstr="--opt-seg-merge", desc="optimal schema for merging ROIs when applying adaptive GTM"
+    ) 
+
+    threads = opt = traits.Int(
+        argstr="--threads %i",
+        desc="threads : number of threads to use"
+    )   
+
+    psf_col = traits.Float(
+        argstr="--psf-col %f",
+        desc="xFWHM : full-width-half-maximum in the x-direction"
+    )  
+
+    psf_row = traits.Float(
+        argstr="--psf-row %f",
+        desc="yFWHM : full-width-half-maximum in the y-direction"
+    )   
+
+    psf_slice = traits.Float(
+        argstr="--psf-slice %f",
+        desc="zFWHM : full-width-half-maximum in the z-direction"
+    )  
+
+class GTMPVCOutputSpec(TraitedSpec):
+
+    pvc_dir = Directory(exists=True, desc="output directory")
+    ref_file = File(exists=True, desc="Reference TAC in .dat")
+    hb_nifti = File(exists=True, desc="High-binding TAC in nifti")
+    hb_dat = File(exists=True, desc="High-binding TAC in .dat")
+    nopvc_file = File(exists=True, desc="TACs for all regions with no PVC")
+    gtm_file = File(exists=True, desc="TACs for all regions with GTM PVC")
+    gtm_stats = File(exists=True, desc="Statistics for the GTM PVC")
+    input_file = File(exists=True, desc="4D PET file in native volume space")
+    reg_pet2anat = File(exists=True, desc="Registration file to go from PET to anat")
+    mgx_ctxgm = File(exists=True, desc="Cortical GM voxel-wise values corrected using the extended Muller-Gartner method")
+    mgx_subctxgm = File(exists=True, desc="Subcortical GM voxel-wise values corrected using the extended Muller-Gartner method")
+    mgx_gm = File(exists=True, desc="All GM voxel-wise values corrected using the extended Muller-Gartner method")
+    rbv = File(exists=True, desc="All GM voxel-wise values corrected using the RBV method")
+    opt_params = File(exists=True, desc="Optimal parameter estimates for the FWHM using adaptive GTM")
+    yhat0 = File(exists=True, desc="4D PET file of signal estimate (yhat) after PVC (unsmoothed)")
+    yhat = File(exists=True, desc="4D PET file of signal estimate (yhat) after PVC (smoothed with PSF)")
+    yhat_full_fov = File(exists=True, desc="4D PET file with full FOV of signal estimate (yhat) after PVC (smoothed with PSF)")
+    yhat_with_noise = File(exists=True, desc="4D PET file with full FOV of signal estimate (yhat) with noise after PVC (smoothed with PSF)")
```

Comment:
I believe the convention is not to use `exists=True` in output specs. @satra could you confirm? I believe this is because it can turn a corrupted working directory into a `TraitsError` instead of simply signaling to nipype that it needs to re-run the node.